docs/about.rst

```diff
@@ -0,0 +1,34 @@
+About PGHoard
+=============
+
+Features
+--------
+
+* Automatic periodic basebackups
+* Automatic transaction log (WAL/xlog) backups (using either ``pg_receivexlog``,
```

Comment:
this should also add pg_receivewal...

Reply:
Replaced it by pg_receivewal.

Reply:
I think replacing is too much, at least until we support pg 9.6 :-/ (when that is EOL, we have a big cleanup in front of us :-/)

Reply:
Ok, I thought it would make sense to refer to the current name of the executable in the about page.
I've updated it again to make sure it is referred as ``pg_receivewal`` (formerly ``pg_receivexlog``) everywhere but in the quickstart, as we don't want that to be confusing.